src/System.Runtime.Extensions/tests/System/AppDomainTests.cs

```diff
@@ -268,28 +316,43 @@ public void IsCompatibilitySwitchSet()
         [Fact]
         public void IsDefaultAppDomain()
         {
-            Assert.True(AppDomain.CurrentDomain.IsDefaultAppDomain());
+            RemoteInvoke(() =>
```

Comment:
yes, I think there is also one more layer of abstraction somewhere since the AppDomainId seems to be 3 on desktop while it is 1 when you run it directly on netfx without any Xunits and other infra